config/foreman.migrations/20160420224417_puppet_autosign.rb

```diff
@@ -2,6 +2,6 @@
 # in puppet-puppet a2325f1 and was deleted from puppet-foreman_proxy 9f3c9aa
 if answers['puppet']
   current_autosign = answers['puppet']['autosign']
-  answers['puppet'].delete('autosign') unless !!current_autosign == current_autosign
+  answers['puppet'].delete('autosign') unless (current_autosign.nil? ? false : current_autoassign.to_bool) == current_autosign
```

Comment:
Would `(current_autosign.nil? || current_autosign.to_bool == current_autosign)` be equivalent?